sr_multi_moveit/sr_multi_moveit_config/scripts/generate_load_moveit_config.py

```diff
@@ -47,7 +47,7 @@
 
     # detect the command to be executed
     if len(sys.argv) > 2:
-        save_file = False
+        save_file = True
```

Comment:
why have you changed this value? Does it influence Jenkins run?

Reply:
We needed to have the files available for IKfast but as it is not part of this pr I turned back False